curvefs/conf/curvebs_client.conf

```diff
@@ -0,0 +1,191 @@
+#
```

Comment:
What should curveadm do for this feature except sync this configure file to the destination?

Reply:
the configuration items that need to be in conf can be configured through curveadm.
configuration items that must be supported: `mds.listen.addr`

Reply:
> the configuration items that need to be in conf can be configured through curveadm. configuration items that must be supported: `mds.listen.addr`

Got it.